src/pages/cli/start/install.mdx

```diff
@@ -64,7 +138,6 @@ This would update/create the AWS Profile in your local machine
 ? Profile Name:  # (default)
 
 Successfully set up the new user.
-```
```

Comment:
Will the callout below work if it's inside the ``` block?

Reply:
This is ending a code block, not starting one